src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -2073,7 +2073,8 @@ private void DecodeWellKnownAttribute(ref DecodeWellKnownAttributeArguments<Attr
             {
                 string verString = (string)attribute.CommonConstructorArguments[0].Value;
                 Version version;
-                if (!VersionHelper.TryParseAssemblyVersion(verString, allowWildcard: true, version: out version))
+                bool deterministic = _compilation.Feature("deterministic")?.Equals("true", StringComparison.OrdinalIgnoreCase) ?? false;
```

Comment:
Doh

`compilation.IsEmitDeterministic` exists.  Don't see how that one slipped by me.